dataikuapi/dss/project.py

```diff
@@ -249,6 +249,29 @@ def set_permissions(self, permissions):
         return self.client._perform_empty(
             "PUT", "/projects/%s/permissions" % self.project_key, body = permissions)
 
+    def get_interest(self):
+        """
+        Get the interest of this project. The interest means the number of watchers and the number of star.
+
+        :return: a dict object containing the project's interest
```

Comment:
Might be useful to add the name of the 2 fields present the dict (starCount and watchCount).